alfresco-webscripts-client/alfresco-webscripts-spring/src/main/java/eu/xenit/alfresco/webscripts/client/spring/http/InsecureSslHttpComponentsClientHttpRequestFactory.java

```diff
@@ -12,9 +12,7 @@
 public class InsecureSslHttpComponentsClientHttpRequestFactory extends HttpComponentsClientHttpRequestFactory {
 
     @SneakyThrows
-    public InsecureSslHttpComponentsClientHttpRequestFactory(HttpClient httpClient) {
-        super(httpClient);
-
+    public InsecureSslHttpComponentsClientHttpRequestFactory(long connectionTimeToLive) {
```

Comment:
it's a bit silly adding a single option here.
Alternatives:
1. pass `HttpProperties` along ?
2. what about allowing a callback with `Consumer<HttpClientBuilder>`, so the caller can configure the http client at will ?

I have a personal preference for (2)